src/ActiveLogin.Authentication.BankId.AspNetCore/Areas/BankIdAuthentication/Views/BankId/_LoginScript.cshtml

```diff
@@ -128,6 +138,21 @@
                 });
         }
 
+        function cancel(requestVerificationToken, returnUrl, loginOptions, orderRef) {
```

Comment:
These parameters are not used and can be removed:
- `returnUrl`
- `loginOptions`

Reply:
Absolutely.